addons/l10n_ar_withholding/tests/test_withholding_ar_ri.py

```diff
@@ -0,0 +1,159 @@
+            'move_type': 'out_invoice',
+            'date': '2023-09-01',
+            'invoice_date': '2023-09-01',
+            'partner_id': cls.partner_a.id,
+            'currency_id': cls.currency_data['currency'].id,
+            'invoice_line_ids': [(0, 0, {'product_id': cls.product_a.id, 'price_unit': 2000.0, 'tax_ids': []})],
+        })
+        
+        
+        cls.product_a.l10n_ar_supplier_withholding_taxes_ids = [Command.set(cls.tax_wth_test_1.ids)]
+        cls.product_b.l10n_ar_supplier_withholding_taxes_ids = [Command.set(cls.tax_wth_test_2.ids + cls.tax_wth_test_1.ids)]
+    
+        cls.l10n_latam_document_type_id = cls.env['l10n_latam.document.type'].search([], limit =1)    
+        cls.tax_21 = cls._search_tax(cls, 'iva_21')
+
+        cls.in_invoice_wht_1 = cls.env['account.move'].create({
+            'move_type': 'in_invoice',
+            'date': '2023-09-01',
+            'invoice_date': '2023-09-01',
+            'partner_id': cls.partner_a.id,
+            'invoice_line_ids': [(0, 0, {'product_id': cls.product_a.id, 'price_unit': 1000.0, 'tax_ids': [Command.set(cls.tax_21.ids)]})],
+            'l10n_latam_document_type_id': cls.l10n_latam_document_type_id.id,
+            'l10n_latam_document_number': '1-1000',
+
+        })
+        cls.in_invoice_wht_1.action_post()
+        cls.in_invoice_wht_2 = cls.env['account.move'].create({
+            'move_type': 'in_invoice',
+            'date': '2023-09-01',
+            'invoice_date': '2023-09-01',
+            'partner_id': cls.partner_a.id,
+            'invoice_line_ids': [(0, 0, {'product_id': cls.product_b.id, 'price_unit': 1000.0, 'tax_ids': [Command.set(cls.tax_21.ids)]})],
+            'l10n_latam_document_type_id': cls.l10n_latam_document_type_id.id,
+            'l10n_latam_document_number': '1-1001',
+        })
+        cls.in_invoice_wht_2.action_post()
+
+        cls.in_invoice_wht_3 = cls.env['account.move'].create({
+            'move_type': 'in_invoice',
+            'date': '2023-09-01',
+            'invoice_date': '2023-09-01',
+            'partner_id': cls.partner_a.id,
+            'invoice_line_ids': [
+                (0, 0, {'product_id': cls.product_b.id, 'price_unit': 1000.0, 'tax_ids': [Command.set(cls.tax_21.ids)]}),
+                (0, 0, {'product_id': cls.product_b.id, 'price_unit': 1000.0, 'tax_ids': [Command.set(cls.tax_21.ids)]})
+            ],
+            'l10n_latam_document_type_id': cls.l10n_latam_document_type_id.id,
+            'l10n_latam_document_number': '1-1002',
+        })
+        cls.in_invoice_wht_3.action_post()
+
+
+    def _search_tax(self, tax_type, type_tax_use='sale'):
+        res = self.env['account.tax'].with_context(active_test=False).search([
+            ('type_tax_use', '=', type_tax_use),
+            ('company_id', '=', self.env.company.id),
+            ('tax_group_id', '=', self.env.ref(f'account.{self.env.company.id}_tax_group_{tax_type}').id)], limit=1)
```

Comment:
What is the point in searching by tax group?

Reply:
the line was removed